specification/resources/resource-manager/Microsoft.Resources/stable/2018-05-01/resources.json

```diff
@@ -2952,6 +2952,11 @@
             "name": "ResourceIdentityType",
             "modelAsString": false
           }
+        },
+        "userAssignedIdentities":{
```

Comment:
This object has a defined shape correct? Can we add the complete schema for this?

Reply:
there is no defined shape
it's a dictionary where keys are the resource ids and the value is an empty jobject

Reply:
Added the example file as discussed

Reply:
Can we have something similar to what compute did https://github.com/Azure/azure-rest-api-specs/blob/6bc56bbc8f9a137a57692bec82cce62ed8579868/specification/compute/resource-manager/Microsoft.Compute/stable/2018-06-01/compute.json#L5642

Reply:
Customer s will have hard time using the SDK without having a shape defined for such fields

Reply:
thanks @anuchandy was having a hard time with how to shape the swagger property